package.json

```diff
@@ -24,5 +24,25 @@
   "bugs": {
     "url": "https://github.com/hicsail/authentication-service/issues"
   },
-  "homepage": "https://github.com/hicsail/authentication-service#readme"
+  "homepage": "https://github.com/hicsail/authentication-service#readme",
+  "devDependencies": {
+    "@types/bcrypt": "^5.0.0",
+    "@types/crypto-js": "^4.1.1",
+    "@types/passport-jwt": "^3.0.7",
+    "@types/randomstring": "^1.1.8",
+    "prisma": "^4.5.0"
+  },
+  "dependencies": {
+    "@nestjs/jwt": "^9.0.0",
```

Comment:
put all these dependencies and devDependencies in the server/package.json rather than top-level package.json